tests/gocase/go.mod

```diff
@@ -7,10 +7,13 @@ require (
 	github.com/stretchr/testify v1.8.0
 )
 
+require github.com/remyoudompheng/bigfft v0.0.0-20200410134404-eec4a21b6bb0 // indirect
```

Comment:
Just curious, why there is a new `require` clause rather than a single `require` with all indirect packages?

Reply:
Oops, maybe it is because that i misplaced the `modernc.org/mathutil v1.5.0` to the indirect packages area... I'll fix both of them.